response-items/src/confirmation/ResponseItem.jsx

```diff
@@ -1,18 +1,22 @@
-import { useEffect, useState } from "react";
+import { useState } from "react";
 import { params } from "./ResponseItem.params";
 import { Flex, Checkbox } from "@chakra-ui/react";
 
 const ResponseItem = ({
   label,
   confirmed: initialChecked,
-  _context: { setNextEnabled, logResults },
+  _context: { setIsValidResponse, logResults, clearResult },
```

Comment:
Just a question, where do `setIsValidResponse` and `clearResult` come from?

Reply:
They are renderContext and Context is designed to share data that can be considered “global” for a tree of React components avoiding the need to be passed along each component tree.
They are declared at surveyPage.jsx